.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: python
+python:
+  - 3.6
+before_install:
+  - python --version
+  - pip install -U pip
+  - pip install -U pytest
+  - pip install codecov
+  - pip install flake8
+install:
+  - pip install -r requirements.txt
+  - pip install -r dev-requirements.txt
+script: 
+    # stop the build if there are Python syntax errors or undefined names
+  - flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics --exclude=harvester_ng/csw,tests/csw,harvester_ng/datajson/tests_with_ckan
+    # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+  - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics --exclude=harvester_ng/csw,tests/csw,harvester_ng/datajson/tests_with_ckan
+
+  - python -m pytest --vcr-record=none tests/datajson
+
+  - safety check --full-report
+  - bandit -c bandit.yml -ll -r .
+after_success:
+  - codecov # submit coverage
+
+deploy:
+    script:
+      - docker build -t viderum/ckan-ng-harvest .
+      - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```

Comment:
where are DOCKER_PASSWORD and DOCKER_USERNAME defined?
They should be added as encrypted env vars in this file.

Reply:
They are defined at [Travis repository settings](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings)

Reply:
Then they should be added as encrypted env vars in this file

Reply:
changed [here](https://github.com/datopian/ckan-ng-harvest/commit/12cdf6d980fdf54e5ba83038d9d8697ef9112827)